cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -160,3 +160,109 @@ def test_vmware_vds_ui_tagging():
             4.You can assign tags to DSwitch.
     """
     pass
+
+
+@pytest.mark.manual()
+@pytest.mark.tier(1)
+def test_vmware_inaccessible_datastore():
+    """
+    VMware sometimes has datastores that are inaccessible, and CloudForms should indicate that.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1684656
```

Comment:
**Required**

This should be:
Bugzilla:
     * 1684656

Reply:
also same comment for the rest below

Reply:
Done